colibris/views.py

```diff
@@ -1,17 +1,195 @@
+import abc
+import traceback
+from json import JSONDecodeError
 
 from aiohttp import web
 from aiohttp_apispec import docs
+from aiohttp_apispec import response_schema, request_schema
 
-from colibris import app
+from colibris.persist import Model
+from colibris.schemas import ModelSchema
+from marshmallow import ValidationError
+from peewee import IntegrityError
+
+from colibris import app, api
 from colibris.conf import settings
 
 
 async def home(request):
     raise web.HTTPFound(settings.API_DOCS_PATH)
 
 
+# TODO: I would suggest to move it into some template.
 @docs(tags=['Service'],
       summary='The health-check endpoint')
 async def health(request):
     h = await app.get_health()
     return web.json_response(h)
+
+
+class _GenericMixinMeta(abc.ABCMeta):
+    def __init__(cls, name, bases, attrs):
+        assert cls.schema_class, f'The "schema_class" field is required for {cls}.'
```

Comment:
Are we ready to ditch Python 3.5? f-strings are supported from 3.6 on, AFAIK.

Reply:
Good catch! I'll change it.